# BMI life expentancy

we'll use linear regression to predict life expectancy from body mass index (BMI).

For our linear regression model, we'll be using scikit-learn's LinearRegression class. This class provides the function fit() to fit the model to our data.

In [2]:
from sklearn.linear_model import LinearRegression

### Load the data

    The data is in the file called "bmi_and_life_expectancy.csv".
    Use pandas read_csv to load the data into a dataframe (don't forget to import pandas!)
    Assign the dataframe to the variable bmi_life_data.



In [7]:
import pandas as pd
bmi_life_data = pd.read_csv("bmi_and_life_expectancy.csv")

### Build a linear regression model

    Create a regression model using scikit-learn's LinearRegression and assign it to bmi_life_model.
    Fit the model to the data.


In [8]:
bmi_life_model = LinearRegression()
bmi_life_model.fit(bmi_life_data[['BMI']], bmi_life_data[['Life expectancy']])

LinearRegression(copy_X=True, fit_intercept=True, n_jobs=1, normalize=False)

### Predict using the model

    Predict using a BMI of "value" and assign it to the variable laos_life_exp.


In [10]:
laos_life_exp  = bmi_life_model.predict(21.07931)
print(laos_life_exp)

[[60.31564716]]


So a person having BMI of 21.07931 will have a life expectancy of 60.31564716